pyanaconda/errors.py

```diff
@@ -271,6 +271,20 @@ def _insightsErrorHandler(self, exn):
         else:
             return ERROR_RAISE
 
+    def _kickstartRegistrationErrorHandler(self, exn):
+        message = _("An error occurred during registration attempt "
+                    "triggered by the rhsm kickstart command. "
+                    "This could have happened due to incorrect rhsm command arguments "
+                    "or subscription infrastructure issues. "
+                    "Would you like to ignore this and continue with "
+                    "installation ?")
```

Comment:
There is an extra space before `?`.

Reply:
An idea from @rvykydal - IMHO it indeed looks better this way.

Reply:
@VladimirSlavik, what do you think?

Reply:
jstodola votes for removing the extra space.

Reply:
Removed. :)